packages/libs/react-components/src/components/Input/styles.ts

```diff
@@ -0,0 +1,130 @@
+/* eslint @kadena-dev/typedef-var: 0 */
+// TODO: Remove this when this issue is resolved: https://github.com/kadena-community/kadena.js/issues/201
+
+import { styled } from '../../styles';
+
+export const styleVariant = {} as const;
+
+export const StyledInputGroupWrapper = styled('div', {
+  color: '$colors$neutral6',
+  '&.disabled': {
+    color: '$colors$neutral3',
```

Comment:
you can just use `$neutral` instead of `$colors$neutral` when its about color values (so color, borderColor, background etc)